website/docs/install-config/upgrade.mdx

```diff
@@ -33,9 +33,9 @@ import Tabs from "@theme/Tabs";
   <TabItem value="docker-compose" label="Docker Compose" default>
     In your terminal, navigate to your installation directory and follow these steps:
 
-    #### 1. Retrieve latest `docker-compose.yml` file
+    #### 1. Retrieve latest `compose.yml` file
 
-    Download the `docker-compose.yml` file using either `wget -O docker-compose.yml https://docs.goauthentik.io/docker-compose.yml` or `curl -O https://docs.goauthentik.io/docker-compose.yml` or a similar process.
```

Comment:
We probably need a PR to github.com/goauthentik/website to serve that file, as well as the old one.

Reply:
I think that the file I'm linking to handles this but I don't fully understand how it works

https://github.com/goauthentik/authentik/pull/16624/files#diff-af888ed9617d810a8c361e2b121dc620fb55f0700f84bb9bfb2c00b406aab68c